src/views/document/OutingView.vue

```diff
@@ -11,7 +11,7 @@
         <tool-box :document="document"/>
       </div>
 
-      <div class="column is-9 is-12-print">
+      <div class="column is-9 is-12-print is-block-print">
```

Comment:
Do you really need `is-block-print`on this one? `column` is a flex child, but does not have `display:flex`

Same question for next changes on other `div.column`

Reply:
Exact, it's 2 errors, the modification is on an other line.

Reply:
Also, if you could has a small comment in helper.scss above ìs-block-print` declaration, with a short explanation, and a link to bugzilla page, it would be perfect.